test/sanitize-html.test.js

```diff
@@ -6,6 +6,30 @@ describe('Sanitize HTML', () => {
     expect(data).toBe('<img src="x" />');
   });
 
+  // note: any sort of image type is accepted using a data URI (.png, .gif, .jpg, etc.)
+  test('<img src="data:..."/> URI links (gif based) should be accepted (i.e. not sanitized)', () => {
+    const data = sanitizeHTML(
+      '<img src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" />'
+    );
+    expect(data).toBe(
+      '<img src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" />'
+    );
+  });
+
+  test('img links over https should be accepted (i.e. not sanitized)', () => {
+    const data = sanitizeHTML(
+      '<p><a><img src="https://www.telescope.com/https_image.jpg" /></a></p>'
+    );
+    expect(data).toBe('<p><a><img src="https://www.telescope.com/https_image.jpg" /></a></p>');
+  });
+
+  test('img links over http should be accepted (i.e. not sanitized)', () => {
```

Comment:
This is an interesting one, since Chrome is moving to block mixed content.  We may need to undo this later, since it will affect our security in the browser (we load over https but mixed content with http images will break us).

Reply:
Neat, should I add a comment specifying this?

Reply:
Sure, maybe link to https://web.dev/what-is-mixed-content/